old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/05-geographic-vs-projected-CRS.Rmd

```diff
@@ -169,22 +189,22 @@ Spatial projection refers to the mathematical calculations
 performed to flatten the 3D data onto a 2D plane (our computer screen
 or a paper map). Projecting data from a round surface onto a flat surface, results
 in visual modifications to the data when plotted on a map. Some areas are stretched
-and some some are compressed. We can see this when we look at a MAP of the entire
+and some some are compressed. We can see this when we look at a map of the entire
 globe.
 
 
 The mathematical calculations used in spatial projections are designed to
 optimize the relative size and shape of a particular region on the globe.
 
 <figure>
-    <a href="http://www.progonos.com/furuti/MapProj/Normal/CartDef/MapDef/Img/devSurfaces.png">
-    <img src="http://www.progonos.com/furuti/MapProj/Normal/CartDef/MapDef/Img/devSurfaces.png">
+    <a href="https://www.earthdatascience.org/images/courses/earth-analytics/spatial-data/spatial-projection-transformations-crs.png">
```

Comment:
@donal-at-NEON: [This site](https://www.earthdatascience.org/courses/earth-analytics/spatial-data-r/geographic-vs-projected-coordinate-reference-systems-UTM/) has the same tutorial! I've been using them for a reference on some of our broken figures links.

Reply:
Oh wow! You might also recognize the author name for that tutorial. Leah Wasser worked at NEON before getting a sweet job at CU (I think she basically founded the EarthLab). Those tutorials are awesome! I should communicate with her and Megan prior to publishing this one, but she developed it on NEON time, so it is rightfully ours to publish as we choose.

Reply:
Awesome! I was wondering why the tutorial was incredibly similar... And I had no idea EarthLab was part of CU. It wasn't established until after I graduated :)